pkgs/development/tools/analysis/binlore/default.nix

```diff
@@ -83,7 +84,7 @@ let
         ((i--)) || true # don't break build
       done # || true # don't break build
       if [[ -d "${drv}/bin" ]] || [[ -d "${drv}/lib" ]] || [[ -d "${drv}/libexec" ]]; then
-        ${pkgsBuildBuild.yara}/bin/yara --scan-list --recursive ${lore.rules} <(printf '%s\n' ${drv}/{bin,lib,libexec}) | ${pkgsBuildBuild.yallback}/bin/yallback ${lore.yallback} "$filter"
+        ${yara}/bin/yara --scan-list --recursive ${lore.rules} <(printf '%s\n' ${drv}/{bin,lib,libexec}) | ${yallback}/bin/yallback ${lore.yallback} "$filter"
```

Comment:
Mainly reverting changes I made in f54875f90e7e32922e1be4599feacdd31cc8833f (which were following on the pattern from 009b9850779197dba8b7e849d303f93be307536f) because I think the need for these is better met by the change to https://github.com/NixOS/nixpkgs/pull/297626/files#diff-1a60ddba374c4b670f65d82294e2ea7449f691de7c35ed0ad43709053316fd2c earlier in this diff.